rinja/src/filters/builtin.rs

```diff
@@ -528,6 +528,215 @@ pub fn title(s: impl fmt::Display) -> Result<String, fmt::Error> {
     Ok(output)
 }
 
+/// For a value of `±1` by default an empty string `""` is returned, otherwise `"s"`.
+///
+/// # Examples
+///
+/// ## With default arguments
+///
+/// ```rust
```

Comment:
You don't need to add `rust` codeblock attribute, it's the default in rust code examples.

Reply:
Oops, you're right. I copied the code from the `linebreaksbr` filter, which had the attribute, too.